packages/sdk/src/instant-trade/InstantTradeBuyerTxBuilder.ts

```diff
@@ -24,18 +26,20 @@ export default class InstantTradeBuyerTxBuilder extends InstantTradeBuilder {
     receiveAddress,
     sellerPSBT,
     feeRate,
-    datasource
+    datasource,
+    outputs
   }: InstantTradeBuyerTxBuilderArgOptions) {
     super({
       address,
       datasource,
       network,
       publicKey,
-      feeRate
+      feeRate,
     })
 
     this.receiveAddress = receiveAddress
     this.decodeSellerPSBT(sellerPSBT)
+    this.outs = outputs ?? []
```

Comment:
let's rename `this.outs` to be consistent with the function naming so we can call this incoming outputs as well. Can keep the params above as `outputs` to be consistent with inscriber class

Reply:
updated `this.outs` to `this.incomingOutputs`